tavern_pathfinder/src/character/mod.rs

```diff
@@ -0,0 +1,68 @@
+use serde::{Serialize,Deserialize};
+use uuid::Uuid;
+
+#[derive(Serialize,Deserialize)]
+pub struct CharacterSummary {
+    char_id: Uuid,
+    name: String,
+    race: Race,
+    level: u8,
+}
+
+#[derive(Serialize,Deserialize)]
+pub struct Character {
```

Comment:
What's the plan for associating a `Character` with one or more levels in a `Class`?